test/recipes/90-test_store.t

```diff
@@ -14,6 +14,32 @@ use OpenSSL::Test qw(:DEFAULT srctop_file srctop_dir bldtop_file data_file);
 my $test_name = "test_store";
 setup($test_name);
 
+# The MSYS2 run-time convert arguments that look like paths when executing
+# a program unless that application is linked with the MSYS run-time.  The
+# exact conversion rules are listed here:
+#
+#       http://www.mingw.org/wiki/Posix_path_conversion
+#
+# With the built in configurations (all having names starting with "mingw"),
+# the openssl application is not linked with the MSYS2 run-time, and therefore,
+# it will receive possibly converted arguments from the process that executes
+# it.  This conversion is fine for normal path arguments, but when those
+# arguments are URIs, they sometimes aren't converted right (typically for
+# URIs without an authority component, 'cause the conversion mechanism doesn't
+# recognise them as URIs) or aren't converted at all (which gives perfectly
+# normal absolute paths from the MSYS viewpoint, but don't work for the
+# Windows run-time we're linked with).
+#
+# It's also possible to avoid conversion by defining MSYS2_ARG_CONV_EXCL with
+# some suitable pattern ("*" to avoid conversions entirely), but that will
+# again give us unconverted paths that don't work with the Windows run-time
+# we're linked with.
+#
+# Checking for both msys perl operating environment and that the target name
+# starts with "mingw", we're doing what we can to assure that other configs
+# that might link openssl.exe with the MSYS run-time are not disturbed.
+my $msys = $^O eq 'msys' && config('target') =~ m|^mingw|;
```

Comment:
Do you mind adding parenthesis? As in my `$msys = (...)` that is.

Reply:
While you are at it, variable name is a bit misleading. mingw_on_msys would be right, but it's somewhat long...

Reply:
I can go for `$mom` if you wish ;-)